### DISCUSSION TITLE
Is there any way to automatically add linenums with snippets? #5345

### DISCUSSION THREAD

**Discussion by HydrogenSulfate** [facelessuser/pymdown-extensions#2006] [Q&A] [answered]
I use material with snippets for some tutorials, I set code blocks in it and set **aligned linenums** with  source code as blow.
![image](https://user-images.githubusercontent.com/23737287/230812498-2ec45980-afea-4f0a-b812-f0b0847253af.png)
But I find it not convinent when I change the source code, I must modify the linenums in codeblocks, so is there any way to automatically keep linenums same as start linenum of a snippet? such as using "auto"?
![image](https://user-images.githubusercontent.com/23737287/230812755-5d802e44-8af5-456e-8566-24206ec4e6f3.png)

same discussion from [squidfunk/mkdocs-material/discussions/5345](https://github.com/squidfunk/mkdocs-material/discussions/5345)

**Comment by facelessuser** [comment, marked as answer]
These two extensions are not interconnected. I don't know how SuperFences would just know it is supposed to start at line 89 without you telling it.
